### DISCUSSION TITLE
'Out of memory' from local NFT marker

### DISCUSSION THREAD

**Discussion by marco2632** [AR-js-org/AR.js#325] [Help]
![Capture](https://user-images.githubusercontent.com/40259446/128660906-d7786f93-fb65-42d4-8ffb-73ca602c4622.PNG)
As the image shown above, I have 3 NFT markers in my application. However, 2 local markers cannot be loaded and showed 'Out of memory' error. The last marker is the trex marker in demo. It works very well. 
The AR application is used within iframe element. When I run the application along, it works. But If I run the whole thing, it shows 'Out of memory'. I don't know how the problem happened and how to fix this problem.

**Comment by Artkoch** [comment]
I get the same error

**Reply by kalwalt** [reply]
This because the size of the WASM memory:
`var MEM = 256 * 1024 * 1024; // 64MB`
https://github.com/AR-js-org/artoolkit5-js/blob/35681ad21d7e4639f81d42429f2acfbf582b6256/scripts/makem.js#L28
is too big, lowering it will solve the issue.
I had the same issue with webarkit/jsartoolkitNFT see issue https://github.com/webarkit/jsartoolkitNFT/issues/97
